nodejs/awsx/utils.ts

```diff
@@ -108,6 +108,26 @@ export function hasTrueBooleanMember(obj: any, memberName: string | number | sym
     return val === true;
 }
 
+/** @internal */
+export function getRegionFromOpts(opts: pulumi.CustomResourceOptions): pulumi.Output<aws.Region> {
+    if (opts.parent) {
+        return getRegion(opts.parent);
+    }
+
+    return getRegionFromProvider(opts.provider);
+}
+
+
+/** @internal */
+export function getRegion(res: pulumi.Resource): pulumi.Output<aws.Region> {
+    return getRegionFromProvider(res.getProvider("aws::"));
+}
+
+function getRegionFromProvider(provider: pulumi.ProviderResource | undefined) {
```

Comment:
Is this also intended to be `@internal`?

Reply:
it's not exported, so i didn't think we needed to mark it one way or hte other.